pkg/metrics/cluster/node.go

```diff
@@ -53,12 +53,13 @@ type node struct {
 // newNode creates a new node and registers it to the ring.
 func newNode(reg prometheus.Registerer, log log.Logger, cfg Config, s pb.ScrapingServiceServer) (*node, error) {
 	n := &node{
-		reg: util.WrapWithUnregisterer(reg),
+		reg: util.WrapWithUnregisterer(prometheus.WrapRegistererWithPrefix("cortex_", reg)),
```

Comment:
Are these equivalent to what it did before?

Reply:
Yes that's correct. You can see Loki made the same changes when they migrated as well: https://github.com/grafana/loki/pull/4641/files#diff-8e51742acf8f88fa2ae01855fb19125feb5c54c41023f1eaa8beded9f1b4bcc3R102

Reply:
cc: @treid314

Reply:
I'm not sure if you want to wrap here since you'll wrap that registerer for all metrics registered with this n.reg with the `cortex_` prefix